components/script/dom/range.rs

```diff
@@ -52,6 +52,12 @@ pub struct Range {
     associated_selections: DomRefCell<Vec<Dom<Selection>>>,
 }
 
+pub struct ContainedChildren {
```

Comment:
I think this type doesn't have to be `pub` since the method `contained_children` is not pub itself and not used outside this file.